jobs/awslogs/templates/config/config/README

```diff
@@ -0,0 +1 @@
+# a place for additional config files as specified in awslogsd.sh
```

Comment:
WDYT about just `mkdir -p` this directory in the pre-start, rather than this placeholder?